### ISSUE TITLE
git describe returns incorrect version

### ISSUE THREAD

**Issue by ingvagabund** [GoogleCloudPlatform/kubernetes#5868]
The latest release is v0.13.2, git describe returns v0.12.0-1145-ged68c8e. How does the master branch relates to the latest release? Is "git describe"'s output intended to be v0.12.0 or is it a bug? This is not the first time inconsistency. Thanks.

**Comment by roberthbailey** [comment]
@zmerlynn Zach tagged that build so maybe he can help here?

**Comment by fabioy** [comment]
I believe Jeff has a PR for this already (PR #5817).

Thanks.

**Comment by zmerlynn** [comment]
The problem here is that despite actually being merged into master, 8d8b9b9a is not reachable from head because of contorted pull gymnastics. You can actually do a `git describe` on c04ceec27 which introduces it, and you'll see:

```
$ git describe c04ceec27f72350b0964e04096ae14c18bee865f
v0.12.0-792-gc04ceec
```

... which is bad news bears. That's because those two commits, 7fbf52a5 and 8d8b9b9a, once they hit master, really don't exist in their present form, so the tag actually only applies to the *branch commit*, as pointed out both here and in #5817.

**Comment by zmerlynn** [comment]
This is all coming about because we changed procedure slightly. The release was based on a good Jenkins run, tagged off there, and then we inject the release number change back into the master branch. I think we need to tag master with something else.

For now, I'm going to tag the branch point as v0.13.0-dev, which may break *other things*, but actually matches the version stamp we put in `pkg/version/base.go`

**Comment by eparis** [comment]
@jbeda has fixed this problem before.  how do we stop it from happening?

**Comment by zmerlynn** [comment]
We have better scripts. I'm looking at it.

**Comment by zmerlynn** [comment]
This actually came about because we changed to a better procedure that allows cranking the release pretty quickly off a clean Jenkins run, rather than just twiddling thumbs waiting on e2e.

**Comment by zmerlynn** [comment]
I pushed a `v0.13.1-dev` tag because @brendanburns v0.13.0 chain was so uniquely messed up that I couldn't reconstruct a useful tag from it.

Closing this.

(BTW, we've never returned the minor version correctly. We typically return the major version, but 0.13.1 was, erm, special and a complete re-release.)
